src/pages/index.tsx

```diff
@@ -39,49 +17,13 @@ const Home: NextPage<TeachersHomePageProps> = (props) => {
     } else if (window.location.href.includes("#ai")) {
       router.push("/ai");
```

Comment:
Does this need to send people to `/` now?

Reply:
i was hoping a redirect inside config would do the job but alas these clientside changes overwrite these. Anyway this is changed now.